Innovator/Client/scripts/kanban/js/script.js

```diff
@@ -235,11 +235,17 @@ function loadBoards() {
 
 		// get boardimage
 		var vaultref = item.getProperty("creatorpicture","undefined");
-		var fileUrl = "nomanager.png";
-		/*if ((vaultref !== "undefined") && (vaultRefTable[vaultref] == undefined)) {
-			fileUrl = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
-			vaultRefTable[vaultref] = fileUrl;
-		}*/
+		var fileUrl = "../kanban/nomanager.png";
+		if ((vaultref !== "undefined") && (vaultRefTable[vaultref] == undefined)) {
+			try{fileUrl = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
+				vaultRefTable[vaultref] = fileUrl;
+				}
+			catch(err){
+				console.log("Profile Image for " + item.getPropertyAttribute("created_by_id", "keyed_name") + " is malformed. Please choose a different image.");
+				}
+			
+		}
+		else{vaultRefTable[vaultref] = fileUrl;}
```

Comment:
The format of our JavaScript code blocks should always look like this:
```
if (condition) {
    // code
} else {
    // code
}
```
Code blocks should not have closing brackets at the end of a line - they should always go on a new line after the block they're closing. This applies to if-else statements, try-catch blocks, and functions.

Reply:
fixed.